src/Entity/ParagraphsRowType.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Drupal\react_paragraphs\Entity;
+
+use Drupal\paragraphs\Entity\ParagraphsType;
+
+/**
+ * Defines the Paragraphs Row type entity.
+ *
+ * @ConfigEntityType(
+ *   id = "paragraphs_row_type",
+ *   label = @Translation("Row type"),
+ *   handlers = {
+ *     "view_builder" = "Drupal\Core\Entity\EntityViewBuilder",
+ *     "list_builder" = "Drupal\react_paragraphs\ParagraphsRowTypeListBuilder",
+ *     "form" = {
+ *       "add" = "Drupal\react_paragraphs\Form\ParagraphsRowTypeForm",
+ *       "edit" = "Drupal\react_paragraphs\Form\ParagraphsRowTypeForm",
+ *       "delete" = "Drupal\react_paragraphs\Form\ParagraphsRowTypeDeleteForm"
+ *     },
+ *     "route_provider" = {
+ *       "html" = "Drupal\react_paragraphs\ParagraphsRowTypeHtmlRouteProvider",
+ *     },
```

Comment:
Are routes needed?

Reply:
not outside of the what the route provider provides